hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1617,7 +1617,7 @@
 
   <property>
     <name>ozone.om.ratis.snapshot.auto.trigger.threshold</name>
-    <value>400000L</value>
+    <value>400000</value>
```

Comment:
Question: If we have taken a snapshot for every 400k, then after that 200k transactions have happened, then when follower OM restart's because it knows it has till 400k only, so will it apply 200k transactions again?

Reply:
Yes that's right

Reply:
Then I think I having less value make sense. 
Do we want to revisit this later, for now just go with ratis default value of 400k?

Reply:
I think 400k should not be too small a number. In HDFS, the default number of transactions after which a checkpoint is saved is 1M. Also, the ratis log index is not the same as the actual transaction count. There are lot of internal ratis log entries also.
But we can re-tweak the default after some testing.

Reply:
Thanks for the info. We can tweak this later.